interface/main/calendar/modules/PostCalendar/pntemplates/default/views/week/ajax_template.html

```diff
@@ -157,10 +157,10 @@
 <input type="hidden" name="jumpdate" id="jumpdate" value="">
 <input type="hidden" name="viewtype" id="viewtype" value="[-php-]echo $viewtype;[-/php-]">
```

Comment:
Brought this script into main codebase (PR is at https://github.com/openemr/openemr/pull/1129)
(note I removed the embedded color black style chevron elements since not needed)